### PR TITLE
Ruby 3.2

### LINKED ISSUE #564: Upgrade to Ruby 3.x

### PR DESCRIPTION
**Pull request by cavis**
Closes #564.

Had to upgrade Feedjira to fix a bunch of date parsing errors (fixed in [feedjira 3.2.2](https://github.com/feedjira/feedjira/blob/main/CHANGELOG.md#322).  Couple of API changes there, but not bad.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -541,7 +537,7 @@ DEPENDENCIES
   webmock
 
 RUBY VERSION
-   ruby 2.7.8p225
+   ruby 3.2.2p53
 
 BUNDLED WITH
-   2.1.4
+   2.4.10
```
Comment (cavis) [inline Gemfile.lock:543]: This is what `ruby:3.2.2-alpine` has installed.  So seemed a good choice!

File: app/models/podcast_import.rb [hunk 2]
```diff
@@ -175,7 +175,7 @@ def create_or_update_episode_import!(entry, has_duplicate_guid = false)
   def get_feed
     response = connection.get(uri.path, uri.query_values)
     self.feed_raw_doc = response.body
-    podcast_feed = Feedjira::Feed.parse(feed_raw_doc)
+    podcast_feed = Feedjira.parse(feed_raw_doc)
```
Comment (cavis) [inline app/models/podcast_import.rb:178]: This was a breaking change in feedjira 3.0.0.

File: config/initializers/feedjira.rb [hunk 3]
```diff
@@ -1,3 +1,5 @@
 load "#{Rails.root}/lib/feedjira/parser/podcast.rb"
 
-Feedjira::Feed.add_feed_class Feedjira::Parser::Podcast
+Feedjira.configure do |config|
+  config.parsers.unshift(Feedjira::Parser::Podcast)
```
Comment (cavis) [inline config/initializers/feedjira.rb:4]: `add_feed_class` no longer exists... but following [the README](https://github.com/feedjira/feedjira/tree/main#parsers)... this "just worked".

I'm not entirely clear if this is a change.  Did `add_feed_class` push the parser on the _end_ of the list, or the beginning?

In any case - the tests still pass.  So... success?

**Review by kookster — APPROVED** [review]
lgtm!

**Pull request merged**